internal/runbits/findproject/project.go

```diff
@@ -68,7 +68,7 @@ func FromNamespaceLocal(ns *project.Namespaced, cfg projectfile.ConfigGetter, pr
 
 	matchingProjects := make(map[string][]string)
 	matchingNamespaces := make([]string, 0)
-	for namespace, paths := range projectfile.GetProjectMapping(cfg) {
+	for namespace, paths := range projectfile.GetStaleProjectMapping(cfg) {
```

Comment:
This doesn't feel right. We might be prompting them to use a project that no longer exists.. that feels very strange.

I don't think we need to be doing this at all. If a user deleted a project then we can't find it. Offering the original location feels like it's just going to cause confusion and buggy code.

In terms of the story I would consider this scenario as "not applicable".

Reply:
Ahh right, I hadn't thought about the prompting. Update this function to store the stale project listing and only use it when we can't find a project locally.